modules/gumgumBidAdapter.js

```diff
@@ -98,8 +98,9 @@ function isBidRequestValid (bid) {
  * @param {validBidRequests[]} - an array of bids
  * @return ServerRequest Info describing the request to the server.
  */
-function buildRequests (validBidRequests) {
+function buildRequests (validBidRequests, bidderRequest) {
   const bids = [];
+  const gdprConsent = Object.assign({ consentString: null, gdprApplies: true }, bidderRequest && bidderRequest.gdprConsent)
```

Comment:
In case when gdpr module does not exist in the build, this will always send `gdprApplies and gdprConsent` to your request payload. 

https://github.com/prebid/Prebid.js/blob/master/modules/appnexusBidAdapter.js#L77